e2e/pages/adf/dialog/shareDialog.ts

```diff
@@ -110,17 +110,16 @@ export class ShareDialog {
         this.timeDatePickerButton.click();
     }
 
-    async calendarTodayDayIsDisabled() {
-        const today = await this.dayPicker.element(by.css('.mat-datetimepicker-calendar-body-today')).getText();
+    calendarTodayDayIsDisabled() {
+        const today = this.dayPicker.element(by.css('.mat-datetimepicker-calendar-body-today')).getText();
         Util.waitUntilElementIsPresent(element(by.cssContainingText('.mat-datetimepicker-calendar-body-disabled', today)));
     }
 
-    async setDefaultDay() {
+    setDefaultDay() {
         const selector = '.mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content';
         Util.waitUntilElementIsVisible(this.dayPicker);
-        const today = await this.dayPicker.element(by.css('.mat-datetimepicker-calendar-body-today')).getText();
-        const tomorrow = (parseInt(today, 10) + 1).toString();
-        this.dayPicker.element(by.cssContainingText(selector, tomorrow )).click();
+        let tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000).getDate().toString();
```

Comment:
might be a bit more clear by using moment
const today = moment();
const tomorrow = today.add(1, 'day');
const dayOfTomorrow = tomorrow.date();